payment/src/main/java/com/kyriba/conference/payment/domain/PaymentEntity.java

```diff
@@ -0,0 +1,76 @@
+package com.kyriba.conference.payment.domain;
+
+import com.kyriba.conference.payment.api.dto.PaymentDto;
+import com.kyriba.conference.payment.api.dto.Receipt;
+import com.kyriba.conference.payment.api.dto.TicketDto;
+import com.kyriba.conference.payment.domain.dto.Amount;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.LocalDateTime;
+import java.util.Currency;
+
+/**
+ * @author Igor Lizura
+ */
+@Entity
+@Table(name = "payment")
+@Getter
+@Setter
+@AllArgsConstructor
```

Comment:
do you use constructor with all args?

Reply:
yes, exactly :)

Reply:
Too many fields for all args constructor. And it is too syntetic, because it real life entity is populated in JPA from defauld constructor and in business logic from DTOs.

Reply:
fixed